src/index.ts

```diff
@@ -102,6 +103,8 @@ function requestToFetchOptions(reqOpts: Options) {
   if (reqOpts.proxy || process.env.HTTP_PROXY || process.env.HTTPS_PROXY) {
     const proxy = (process.env.HTTP_PROXY || process.env.HTTPS_PROXY)!;
     options.agent = new HttpsProxyAgent(proxy);
+  } else if (reqOpts.forever) {
+    options.agent = new Agent({keepAlive: true});
```

Comment:
A test would be lovely :)

Reply:
You got it! Just wanted to make sure the implementation looked ok first

Reply:
@bcoe @JustinBeckwith the tests here are a little lacking and there isn't really a precedent for testing that request options get mapped to fetch options correctly. My gut says to use something like `proxyquire` to intercept the request and check the supplied options passed to `fetch`. Any feelings here?